packages/spanner/system-test/spanner.js

```diff
@@ -108,6 +108,22 @@ var spanner = new Spanner(env);
       }, execAfterOperationComplete(done));
     });
 
+    describe('uneven rows', function() {
+      it('should allow inserting uneven rows', function(done) {
+        table.insert([
+          {
+            Key: generateName('id'),
+            BoolValue: true
+          },
+          {
+            Key: generateName('id'),
+            BoolValue: true,
+            IntValue: 10
+          }
+        ], done);
```

Comment:
I don't think so. The issue mentioned that even explicitly passing null can cause problem. Also we should throw an error if number of values in each array is not same.